nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -191,23 +189,17 @@ def get_spec(self):
                     usage="download_binary file_name ...",
                     handler_func=self.download_binary_file,
                 ),
-                CommandSpec(
-                    name=self.upload_folder_cmd_name,
-                    description="upload application to the server",
-                    usage=self.upload_folder_cmd_name + " application_folder",
-                    handler_func=self.upload_folder,
-                ),
                 CommandSpec(
                     name="download_folder",
                     description="download a folder from the server",
                     usage="download_folder folder_name",
                     handler_func=self.download_folder,
                 ),
                 CommandSpec(
-                    name="upload_job",
-                    description="upload application to the server",
-                    usage="upload_job job_folder",
-                    handler_func=self.upload_job,
+                    name="submit_job",
+                    description="Submit application to the server",
+                    usage="submit_job job_folder",
+                    handler_func=self.submit_job,
```

Comment:
The "submit_job" command is missing the "authz_func=...". We used to have separate steps of "upload_folder", then "deploy_app" and "start_app" commands to run the App. "deploy_app" and "start_app" have the authorization check. Now the "submit_job" command makes this into one step command. We need to build the authz_func for this command.

BTW, this command should be moved to the job_cmds.py now with this function change.

Reply:
Agree. Will do it in next PR.